doc/uftrace-report.md

```diff
@@ -223,6 +213,59 @@ The example below will sort output by total time of the base data.
         0.920 us    0.789 us   -14.24%     0.920 us    0.789 us   -14.24%            1          1         +0   __cxa_atexit
         0.767 us    0.706 us    -7.95%     0.767 us    0.706 us    -7.95%            1          1         +0   getpid
 
+FIELDS
+======
+The uftrace allows for user to customize the report output with a couple of
+fields.  Here the field means info on the left side of the pipe (|) character.
```

Comment:
There's no pipe in the report output unfortunately. :)

Reply:
oops. thank you.
I hope to fix it with other modification after clarifying specification.

Reply:
ok, good.